packages/grafana-schema/src/raw/composable/bargauge/panelcfg/x/BarGaugePanelCfg_types.gen.ts

```diff
@@ -15,18 +15,22 @@ export const pluginVersion = "10.3.0-pre";
 
 export interface Options extends common.SingleStatBaseOptions {
   displayMode: common.BarGaugeDisplayMode;
+  maxVizHeight: number;
   minVizHeight: number;
   minVizWidth: number;
   namePlacement: common.BarGaugeNamePlacement;
   showUnfilled: boolean;
+  sizing: common.BarGaugeSizing;
   valueMode: common.BarGaugeValueMode;
 }
 
 export const defaultOptions: Partial<Options> = {
   displayMode: common.BarGaugeDisplayMode.Gradient,
-  minVizHeight: 10,
-  minVizWidth: 0,
+  maxVizHeight: 300,
+  minVizHeight: 75,
+  minVizWidth: 75,
```

Comment:
@nmarrs @Develer these changes sadly these changes to the defaults that are still used in "auto" mode completely and utterly break the auto mode. The sad part is that we cannot just restore the values to fix it as any saved dashboard will be saved with these new values. 

but maybe the fix is just to not use these defaults in auto mode.

Reply:
wait, no we can just change these. even though they are persisted the defaults are directly used when sizing is set to auto